rasa/core/featurizers.py

```diff
@@ -133,6 +137,76 @@ def create_encoded_all_actions(self, domain: Domain) -> np.ndarray:
         return np.eye(domain.num_actions)
 
 
+class BOWSingleStateFeaturizer(CountVectorsFeaturizer, SingleStateFeaturizer):
+    def prepare_training_data_and_train(self, trackers_as_states, delimiter="_"):
```

Comment:
could you please add a TODO that for real text data there will be no delimiter, and we'll need to think about utilizing whole nlu pipeline. But for now current implementation should be enough